### ISSUE TITLE
Macros whose first line starts with =head1 fail to load.

### ISSUE THREAD

**Issue by somiaj** [openwebwork/pg#961]
Loading a macro with `loadMacros` whose first line starts POD, such as `=head1 NAME`, fail to load with errors as the POD delimiter is ignored, and the text is read as code which fails. If a blank line or comment is added to the start of the macro, it loads just fine. This wasn't an issue in 2.18. For example, saving the following as `testMacro.pl` and loading it fails.

```perl
=head1 NAME
This is a test
=cut
$tau = Real("2pi");
```

But putting a comment on the first line, the macro loads just fine.

```perl
# Comment
=head1 NAME
This is a test
=cut
$tau = Real("2pi");
```

**Comment by drgrice1** [comment]
In my opinion this is not an issue.  That is actually what is expected by Perl POD.  Any of the POD directives (lines that start with `=`) should have and empty line before and after them.

This is caused because I removed the newline from the eval call.  You will need to take your pick of what is more important.  Do you want errors and warning reported at the correct line number in the macro file, or do you want this to work?  In my opinion, this behavior is not important (particularly since it is invalid POD in any case).

**Comment by drgrice1** [comment]
Note also that if you run `perltidy` on a Perl file that starts with a POD directive with no empty line before it (like `=head`), then it will add an empty line.  You should run `perltidy` on your macro files anyway.

**Comment by somiaj** [comment]
I'm okay with this not being a real issue due to I'm using invalid POD. I just had multiple macros stop working for me that did work in 2.18 because I didn't realize there had to be a new line and it took me a while to figure out what the actual problem was with the macro files (the errors were strange but didn't actually point me to the issue with the POD).  I may not be the only person who has custom macro files like this that may break in the upgrade from 2.18.

**Comment by dlglin** [comment]
I added a note to the [release notes](https://webwork.maa.org/wiki/Release_notes_for_WeBWorK_2.19).

**Comment by drgrice1** [comment]
Note that with pull request #972 the macro will compile, but a warning will be emitted.

**Comment by somiaj** [comment]
Fixed with #962.
